go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/Azure/azure-storage-azcopy
 require (
 	cloud.google.com/go/storage v1.8.0
 	github.com/Azure/azure-pipeline-go v0.2.3
-	github.com/Azure/azure-storage-blob-go v0.10.1-0.20210407023846-16cf969ec1c3
-	github.com/Azure/azure-storage-file-go v0.6.1-0.20201111053559-3c1754dc00a5
+	github.com/Azure/azure-storage-blob-go v0.10.1-0.20210325045028-fa5d89d7d3c9
```

Comment:
@siminsavani-msft ,

We need `github.com/Azure/azure-storage-blob-go v0.10.1-0.20210407023846-16cf969ec1c3` instead of `github.com/Azure/azure-storage-blob-go v0.10.1-0.20210325045028-fa5d89d7d3c9` as the later is an older version.

Please revert this line.